actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -49,6 +50,23 @@ class TagBuilder # :nodoc:
         HTML_VOID_ELEMENTS = %i(area base br col embed hr img input keygen link meta param source track wbr).to_set
         SVG_SELF_CLOSING_ELEMENTS = %i(animate animateMotion animateTransform circle ellipse line path polygon polyline rect set stop use view).to_set
 
+        def self.define_void_element(name, self_closing: false)
+          class_eval <<~RUBY, __FILE__, __LINE__ + 1
+            def #{name}(content = nil, escape: true, **options, &block)
+              if content.nil? && block.nil?
+                void_tag_string(#{name.inspect}, options, escape, #{self_closing})
+              else
+                tag_string(#{name.inspect}, content, escape: escape, **options, &block)
+              end
+            end
+          RUBY
+
+          alias_method name.to_s.underscore, name unless name.to_s == name.to_s.underscore
```

Comment:
Why this alias?

Reply:
Ahh, got it. Yeah. I don't think we should define a method called `animateTransform`. That isn't Ruby. Let's only define the `animate_transform` version.